src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,55 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Block {
+	
+	public static final String EXAMPLE = "123";
+    public static final String MESSAGE_BLOCK_CONSTRAINTS = "Block can be in any format";
+    public static final String BLOCK_VALIDATION_REGEX = ".+";
+    
+    public final String value;
+    private boolean isPrivate;
```

Comment:
Think about what can we make use of (or how can we make use of) this `isPrivate` variable`.